config/services.yaml

```diff
@@ -32,10 +32,6 @@ services:
         resource: '../src/Presentation/Api/Handler'
         tags: ['controller.service_arguments']
 
-    App\Presentation\Api\Grower\DataTransformer\:
-        resource: '../src/Presentation/Api/Grower/DataTransformer'
-        tags: ['api_platform.data_transformer']
-
```

Comment:
This lines can be deleted

Reply:
This lines will be deleted for the next commit. ;)